generated/nidcpower/nidcpower/_library_singleton.py

```diff
@@ -51,5 +51,17 @@ def get():
             except OSError:
                 raise errors.DriverNotInstalledError()
             _instance = _library.Library(ctypes_library)
+            try:
+                runtime_env_ctype = ctypes.create_string_buffer(platform.python_implementation().encode('windows-1251'))  # case C010
```

Comment:
I think it's unlikely that the Python implementation and version will contain Cyrillic characters. Consider using ascii, or at least windows-1252 (Latin 1).